api/src/middleware/cors.ts

```diff
@@ -6,7 +6,7 @@ let corsMiddleware: RequestHandler = (req, res, next) => next();
 
 if (env.CORS_ENABLED === true) {
 	corsMiddleware = cors({
-		origin: env.CORS_ORIGIN || true,
+		origin: env.CORS_ORIGIN_REGEXP === true ? new RegExp(env.CORS_ORIGIN) : env.CORS_ORIGIN || true,
```

Comment:
What do you think of using this logic instead of an .env variable?
https://github.com/expressjs/cors/blob/5c0b6c7a0cbf126c949b9a76c7c67e26eba6b3e1/lib/index.js#L29

Reply:
As the environment variable, it is always a string, it would always be string value.